README.md

```diff
@@ -4,14 +4,49 @@
 [![Build Status](https://github.com/software-gardening/almanack/actions/workflows/pytest-tests.yml/badge.svg?branch=main)](https://github.com/software-gardening/almanack/actions/workflows/pytest-tests.yml?query=branch%3Amain)
 ![Coverage Status](https://raw.githubusercontent.com/software-gardening/almanack/main/media/coverage-badge.svg)
 
-An open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+The Software Gardening Almanack is an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+The Almanack is available both as a book and a Python package.
```

Comment:
Not sure if this sounds the best, but might be a good idea to keep the terminology consistent 
```suggestion
The Almanack is available both as a handbook and a Python package.
```

Reply:
Thanks, yes, that follows the existing language use and I like the way it sounds!